Bottom: f9546345078ec23c6d5de14df6ef01747ff56425
Top:    f48743f2a87de10d58d9554fb69202b22d949ec6
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-16 14:06:10 -0500

Create certificate directories during image build



---
diff --git a/docker/docker-mediawiki/configure.sh b/docker/docker-mediawiki/configure.sh
index 06ed2991..eba1977c 100755
--- a/docker/docker-mediawiki/configure.sh
+++ b/docker/docker-mediawiki/configure.sh
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -e
 
+# Create certificate directories
+mkdir -p /etc/grid-security/certificates
+mkdir -p /etc/ssl/certs/
+mkdir -p /etc/ssl/private
+
 # comment out apache2 config file lines that reference the environment variables
 sed -i 's|^Mutex file|#Mutex file|' /etc/apache2/apache2.conf
 sed -i 's|^PidFile |#PidFile |' /etc/apache2/apache2.conf
